### PR TITLE
Cleans public on every build

### PR DESCRIPTION
**Pull request by akushniruk**

### CODE REVIEW

File: skel/README.md [hunk 1]
```diff
@@ -4,16 +4,36 @@ This application was generated with [sonic](https://github.com/openware/sonic)
...
+3. Create db
+```
+docker-compose -f config/backend.yml exec db bash
+mysql
+create database opendax_development;
+exit
+exit
+```
+
+4. Set up vault
```
Comment (dpatsora) [inline skel/README.md:28]: Do we need this already?

File: skel/README.md [hunk 2]
```diff
@@ -4,16 +4,36 @@ This application was generated with [sonic](https://github.com/openware/sonic)
...
-#### **Be sure that you build your client (frontend) to the build folder, if it's another folder you can update your client (frontend) or makefile**
+It will run build in the web folder and then all build files will be moved to public/assets.
+#### **Be sure that you build your web (frontend) to the build folder, if it's another folder you can update your client (frontend) or makefile**
+
+2. Start db, redis and vault
+```
+docker-compose -f config/backend.yml up -Vd
+```
+
+3. Create db
```
Comment (dpatsora) [inline skel/README.md:19]: No need, because `opendax_development ` db exists by default due to compose file

File: skel/README.md [hunk 3]
```diff
@@ -4,16 +4,36 @@ This application was generated with [sonic](https://github.com/openware/sonic)
 
 # How to run sonic with client
 
-1. Put your frontend application to the client folder
```
Comment (dpatsora) [inline skel/README.md:7]: Maybe don't remove this ?

**Review by dpatsora — CHANGES_REQUESTED** [review]

File: skel/README.md (hunk 3 above)
Comment (akushniruk) [reply skel/README.md:7]: + i will put it back

**Pull request closed without merging**